GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -1081,5 +1075,40 @@ public static void TerminateTree(this Process process)
                 process.Kill();
             }
         }
+
+        public static string FormatCommitMessageFromTextBox(
+            string commitMessageText, bool usingCommitTemplate, bool ensureCommitMessageSecondLineEmpty)
+        {
+            if (commitMessageText == null)
```

Comment:
What about `commitMessageText == ""` or `commitMessageText == "    "`?

Reply:
This is a pre-existing code. If replacing the above with `IsNullOrWhitespace()` is not going to affect the existing logic, I can push the change.